layouts/shortcodes/observability_pipelines/processors/filter_syntax.md

```diff
@@ -11,6 +11,7 @@ Filter query examples:
 - `status:ok service:flask-web-app`: This filters for all logs with the status `OK` from your `flask-web-app` service.
     - This query can also be written as: `status:ok AND service:flask-web-app`.
 - `host:COMP-A9JNGYK OR host:COMP-J58KAS`: This filter query only matches logs from the labeled hosts.
+- `@user.status:inactive`: This filters for logs with the key-value attribute `status:inactive` nested under the `user` attribute.
```

Comment:
🤔  this might be nit picky but when I read "attribute", I read it as synonymous with "key". So the text "key-value attribute" reads a bit confusing. I might suggest something along the lines of

```suggestion
- `@user.status:inactive`: This filters for logs with the status `inactive` nested under the `user` attribute. 
```

, That also has more synergy with the line two lines above about flask web app. WDYT ?